src/init.ts

```diff
@@ -658,75 +659,77 @@ export class NightwatchInit {
+        Logger.info('These can be located at:');
+        Logger.info(
           `{\n  ...\n  "test_settings": {\n    ...\n    "${answers.remoteName}": {\n      "selenium": {\n        ${colors.cyan(
             '"host":')}\n        ${colors.cyan('"port":')}\n      }\n    }\n  }\n}`,
           '\n'
         );
 
-        Logger.error(
+        Logger.info(
           'Please set the credentials (if any) required to run tests on your cloud provider or remote selenium-server, by setting the below env variables:'
         );
       } else {
-        Logger.error(
+        Logger.info(
           'Please set the credentials required to run tests on your cloud provider, by setting the below env variables:'
         );
       }
 
-      Logger.error(`- ${colors.cyan(answers.remoteEnv?.username as string)}`);
-      Logger.error(`- ${colors.cyan(answers.remoteEnv?.access_key as string)}`);
-      Logger.error('(.env files are also supported)', '\n');
+      Logger.info(`- ${colors.cyan(answers.remoteEnv?.username as string)}`);
+      Logger.info(`- ${colors.cyan(answers.remoteEnv?.access_key as string)}`);
+      Logger.info('(.env files are also supported)', '\n');
     }
-    Logger.error();
+    Logger.info();
 
     const relativeToRootDir = path.relative(process.cwd(), this.rootDir) || '.';
 
     // For now the templates added only for JS
     if (answers.runner !== Runner.Cucumber && answers.language !== 'ts') {
-      Logger.error(colors.green('TEMPLATE TESTS'), '\n');
-      Logger.error('To get started, checkout the following templates. Skip/delete them if you are an experienced user.');
-      Logger.error(colors.cyan(`  1. Title Assertion (${path.join(relativeToRootDir, answers.examplesLocation || '', 'templates', 'titleAssertion.js')})`));
-      Logger.error(colors.cyan(`  2. Login (${path.join(relativeToRootDir, answers.examplesLocation || '', 'templates', 'login.js')})`));
-      Logger.error();
+      Logger.info(colors.green('📃 TEMPLATE TESTS'), '\n');
+      Logger.info('To get started, checkout the following templates. Skip/delete them if you are an experienced user.');
+      Logger.info(colors.cyan(`  1. Title Assertion (${path.join(relativeToRootDir, answers.examplesLocation || '', 'templates', 'titleAssertion.js')})`));
+      Logger.info(colors.cyan(`  2. Login (${path.join(relativeToRootDir, answers.examplesLocation || '', 'templates', 'login.js')})`));
+      Logger.info();
     }
   
     let configFlag = '';
     if (this.otherInfo.nonDefaultConfigName) {
       configFlag = ` --config ${this.otherInfo.nonDefaultConfigName}`;
     }
 
+    Logger.info(colors.green('✨ SETUP COMPLETE'));
+    execSync('npx nightwatch --version', {
+      stdio: 'inherit',
+      cwd: this.rootDir
+    });
+
+    // Join Discord and GitHub
+    Logger.info('💬 Join our Discord community to find answers to your issues or queries. Or just join and say hi.');
+    Logger.info(colors.cyan('   https://discord.gg/SN8Da2X'), '\n');
```

Comment:
Skipping GitHub is intentional here? Since the comment above still mentions GitHub.

Reply:
Yes, displaying the Github link adds no real value to the output.